### PR TITLE
[master-2.x]Add MaxConcurrentConnections in config.json

### PR DESCRIPTION
**Pull request by superboyiii**
Need [neo#1887](https://github.com/neo-project/neo/pull/1873)

### CODE REVIEW

**Review by shargon — APPROVED** [review]
Wait for https://github.com/neo-project/neo/pull/1873

**Review by shargon — APPROVED** [review]

**Comment by superboyiii** [comment]
@shargon A little mistake, I've just fixed it.

**Review by shargon — APPROVED** [review]

**Comment by shargon** [comment]
https://github.com/neo-project/neo/pull/1887 was merged, @superboyiii could you update the nuget ?

**Comment by superboyiii** [comment]
> [neo-project/neo#1887](https://github.com/neo-project/neo/pull/1887) was merged, @superboyiii could you update the nuget ?

Sure.

**Review by erikzhang — CHANGES_REQUESTED** [review]
The configuration files and the default value must be consistent.

**Comment by superboyiii** [comment]
> The configuration files and the default value must be consistent.

Done

File: neo-cli/Settings.cs [hunk 1]
```diff
@@ -66,15 +66,17 @@ internal class RPCSettings
...
 
         public RPCSettings(IConfigurationSection section)
         {
             this.BindAddress = IPAddress.Parse(section.GetSection("BindAddress").Value);
             this.Port = ushort.Parse(section.GetSection("Port").Value);
             this.SslCert = section.GetSection("SslCert").Value;
             this.SslCertPassword = section.GetSection("SslCertPassword").Value;
-            this.MaxGasInvoke = Fixed8.Parse(section.GetValue("MaxGasInvoke", "0"));
+            this.ExtraGasInvoke = Fixed8.Parse(section.GetValue("ExtraGasInvoke", "0"));
+            this.MaxConcurrentConnections = int.Parse(section.GetValue("MaxConcurrentConnections", "120"));
```
Comment (erikzhang) [inline neo-cli/Settings.cs:79]: `120` is too high for normal use.

```suggestion
            this.MaxConcurrentConnections = int.Parse(section.GetValue("MaxConcurrentConnections", "10"));
```

Comment (superboyiii) [reply neo-cli/Settings.cs:79]: > MaxConcurrentConnections

`10` concurrency is too low for a full node rpc. Mainnet seed nodes is meeting more than `40` concurrency now. And rising it to this amount doesn't influence node stability or performance.

Comment (erikzhang) [reply neo-cli/Settings.cs:79]: You can change the configuration file for seed nodes. But the default settings of the configuration files should suit the needs of normal users.

Comment (superboyiii) [reply neo-cli/Settings.cs:79]: > You can change the configuration file for seed nodes. But the default settings of the configuration files should suit the needs of normal users.

Fine.

**Review by erikzhang — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**